client/wsclient.go

```diff
@@ -80,13 +104,53 @@ func (c *wsClient) Start(ctx context.Context, settings types.StartSettings) erro
 }
 
 func (c *wsClient) Stop(ctx context.Context) error {
+	c.isStopped.Store(true)
```

Comment:
Can we rely on the `isStoppingFlag` from ClientCommon? I think the intent behind that flag and this is the same.

Reply:
Ah, I understand the complication it brings. This is fine.